packages/pn-commons/package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "start": "tsc -w --sourceMap=true",
     "build": "tsc",
-    "test": "react-scripts test --watchAll=false",
+    "test": "tsc && react-scripts test --watchAll=false",
```

Comment:
because after pull request i have an error with control pass. 
in the project they passed the tests but not the sonarlint control, this is because with the test command, the code is not compiled

Reply:
I verified with @maurizioflauti and @LorenzoCocciarelli , the added `tsc` is not needed. I just pushed a commit having erased again the `tsc &&`, it passes all the sonarlint controls.
I guess that the error in a previous run was related to a code error, cfr. the following image from the logs.
![image](https://user-images.githubusercontent.com/5519535/208171192-af909881-cf6e-4c42-9d8b-4b851decc1a1.png)